cmd/server/command/start.go

```diff
@@ -131,11 +133,29 @@ func NewStart(startOptions *startOptions) *cobra.Command {
 					},
 				},
 			}
+			// Import GPG Keys
+			if startOptions.gitConfigOpts.SigningKey != "" {
+				if len(startOptions.gitConfigOpts.GPGImportPaths) < 1 {
+					return errors.New("gpg signing key provided, but no import paths specified")
+				}
+				for _, p := range startOptions.gitConfigOpts.GPGImportPaths {
+					// lets just use flux' implementation on how to load keys
+					keyfiles, err := gpg.ImportKeys(p, false)
+					if err != nil {
+						return errors.New(fmt.Sprintf("failed to import GPG key(s) from %s", p))
+					}
+					if keyfiles != nil {
+						log.Infof("imported GPG key(s) from %s files %v", p, keyfiles)
```

Comment:
I don't understand this log line.
It will be something like `imported GPG key(s) from /a/path files [a.key b.key]`?

Reply:
It just indicates that gpg keys from the specified paths have been imported and output their filenames.